### PR TITLE
[tool] Upgraded RegressionChecker toolkit

### PR DESCRIPTION
**Pull request by BruceDai**
@ibelem PTAL, Thanks.
This PR is to:
1. Updated baseline data by 73f3526.       
2. Modified package.json, since latest chromedrive just supports chrome 71+, set chromedriver version as 2.45.0 to support current chromium of 70.

### CODE REVIEW

File: test/tools/RegressionChecker/package.json [hunk 1]
```diff
@@ -12,8 +12,19 @@
...
     "selenium-webdriver": "^4.0.0-alpha.1"
-  }
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/01org/web-test-suite/RegressionChecker.git"
+  },
+  "keywords": [],
+  "author": "yanx.cui@intel.com",
+  "license": "Apache 2.0",
```
Comment (ibelem) [inline test/tools/RegressionChecker/package.json]: Why do we need this?

File: test/tools/RegressionChecker/package.json [hunk 2]
```diff
@@ -12,8 +12,19 @@
...
   "devDependencies": {
-    "chromedriver": "^2.41.0",
+    "chromedriver": "2.45.0",
     "fast-csv": "^2.4.1",
     "selenium-webdriver": "^4.0.0-alpha.1"
-  }
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/01org/web-test-suite/RegressionChecker.git"
```
Comment (ibelem) [inline test/tools/RegressionChecker/package.json]: Why do we need this?

File: test/tools/RegressionChecker/package.json [hunk 3]
```diff
@@ -12,8 +12,19 @@
...
+  },
+  "keywords": [],
+  "author": "yanx.cui@intel.com",
```
Comment (ibelem) [inline test/tools/RegressionChecker/package.json]: Why do we need this?

File: test/tools/RegressionChecker/package.json [hunk 4]
```diff
@@ -12,8 +12,19 @@
...
+  "license": "Apache 2.0",
+  "bugs": {
+    "url": "https://github.com/01org/web-test-suite/RegressionChecker/issues"
```
Comment (ibelem) [inline test/tools/RegressionChecker/package.json]: Please remove this section as well.

Comment (BruceDai) [reply test/tools/RegressionChecker/package.json]: @ibelem Done, thanks

**Comment by ibelem** [comment]
LGTM, thanks a lot! :)

**Pull request merged**